cmd/syncthing/gui.go

```diff
@@ -1017,6 +1020,30 @@ func (s *apiService) getEvents(w http.ResponseWriter, r *http.Request) {
 	sendJSON(w, evs)
 }
 
+func (s *apiService) getDiskEvents(w http.ResponseWriter, r *http.Request) {
+	qs := r.URL.Query()
+	sinceStr := qs.Get("since")
+	limitStr := qs.Get("limit")
+	since, _ := strconv.Atoi(sinceStr)
+	limit, _ := strconv.Atoi(limitStr)
+
+	s.fss.gotEventRequest()
```

Comment:
Not needed here, as listening to these events doesn't imply an interest in the folder summary events.